c10/core/Device.cpp

```diff
@@ -13,7 +13,7 @@
 namespace c10 {
 namespace {
 DeviceType parse_type(const std::string& device_string) {
-  static const std::array<std::pair<std::string, DeviceType>, 9> types = {{
+  static const std::array<std::pair<std::string, DeviceType>, 10> types = {{
```

Comment:
Umm... but you didn't actually update the array?  Also, I thought quantized wasn't a device type? (So I guess these are some old changes that failed to get reverted?)

Reply:
Yeah, will remove